test/utils/db_utils.py

```diff
@@ -0,0 +1,20 @@
+import textwrap
+
+from pyexasol import ExaConnection
+
+
+def create_schema(pyexasol_connection: ExaConnection, schema: str):
+    pyexasol_connection.execute(f"DROP SCHEMA IF EXISTS {schema} CASCADE;")
+    pyexasol_connection.execute(f"CREATE SCHEMA IF NOT EXISTS {schema};")
+
+
+def assert_udf_running(pyexasol_connection: ExaConnection, language_alias: str, schema: str):
```

Comment:
Asserts in non-test files don't get rewritten by pytest, you need to use `pytest.register_assert_rewrite("utils.db_utils")` in the conftest.py, I think.

https://docs.pytest.org/en/7.1.x/how-to/writing_plugins.html#assertion-rewriting